samples/tls/logging/README.md

```diff
@@ -37,9 +36,51 @@ After you finish modifying the customization files, deploy logging using the sta
 ```bash
 my_repository_path/logging/bin/deploy_logging_open.sh
 ```
-## Notes On Customization Values
+## Specifying TLS for Connections to Kibana
 
 Specify `TLS_ENABLE=true` in the `user.env` file to require TLS for connections to Kibana. Connections to Elasticsearch (when enabled) always use TLS, regardless of the value of `TLS_ENABLE`.
 
+If you specify `TLS_ENABLE=true`, the `kibana-tls-secret` Kubernetes secret 
+must be present. By default, the `deploy_logging_open.sh` deployment script automatically obtains the certificates from cert-manager and creates 
+the `kibana-tls-secret` secret.
+
+To create the secret manually, issue this command:
+```bash
+kubectl -n <namespace> create secret generic <secret-name> --from-file=tls.crt=<tls_cert_name>.pem --from-file=tls.key=<key_name>.key --from-file=ca.crt=<CA_key_name>.pem
+```
+By default, the value of `namespace` that is used during the deployment process is `logging`. Specify `kibana-tls-secret` as the value for `secret-name`.
+
+## Specifying TLS for Ingress
+
+If you are using ingress and want to enable TLS on traffic into the cluster, 
+you must also manually create two Kubernetes secrets.
+
+1. Issue this command to generate the required secrets:
+
+```bash
+kubectl -n <namespace> create secret generic <secret-name> --from-file=tls.crt=<tls_cert_name>.pem --from-file=tls.key=<key_name>.key --from-file=ca.crt=<CA_key_name>.pem
+```
+By default, the value of `namespace` that is used during the deployment process is `logging`. Run the command for for each of these values of `secret-name`:
+
+- `elasticsearch-ingress-tls-secret`
+- `kibana-ingress-tls-secret`
+
+Use the appropriate values for `tls_cert_name`, `key_name`, and `CA_key_name` for each secret that that is being generated.
+
+2. After you have obtained the certificates for the secrets, use this command to generate the secrets:
+
+```bash
+kubectl create secret tls $SECRET_NAME -n $NAMESPACE --key $CERT_KEY --cert $CERT_FILE
+```
+
+Use `elasticsearch-ingress-tls-secret` and `kibana-ingress-tls-secret` as values for `$SECRET_NAME`. Use the name of the namespace into which the logging components 
+were deployed (such as `logging`) for the value of `$NAMESPACE`.
+
```

Comment:
Steps 1 and 2 are doing the same thing (i.e. creating the secret from the certs).  I notice the syntax is different...go with what is there in Step 2 since that is pulled (I''m pretty sure) from the original version of this particular doc.  
Since the syntax used in the previous section is different, it might confuse the users.  I suggest rather than showing it here, you just point them to the NOTES ON TLS doc for the details of creating the kibana-tls-secret secret if they want to do that manually.